rust/wallet_core_rs/src/ffi/ethereum/abi.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright © 2017-2023 Trust Wallet.
```

Comment:
None of the function in this file can return an Error?

Reply:
They can, but in most cases the error is set at the corresponding Protobuf message.
For example, `tw_ethereum_abi_decode_contract_call` returns serialized `ContractCallDecodingOutput` message, and `ContractCallDecodingOutput::error` indicates if there is error on decoding a contract call.

But sometimes `tw_ethereum_abi_decode_contract_call` can return a null pointer if:
1. `input` pointer is null
2. `coin` type is unsupported, so `evm_dispatcher` function returned `None`